index.html

```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+    <section class="5-page">
+        <div class="snapshot2">
+            <a href=""><img src="Pictures/snapshot4.png" alt=""></a>
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+
+
+    <section class="aboutme">
+        <div class="headline">
+            <div class="headline-heading">
+                <h1>About myself</h1>
+            </div>
+            <div class="headline-bio">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+            </div>
+            <div class="lets-connect">
+                <h5>LET'S CONNECT</h5>
+            </div>
+            <div class="icons">
+                <ul>
+                    <li><a href="https://github.com/Boku52500"><img src="Pictures/github.png" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
+                                src="Pictures/Linkedin.png" alt=""></a></li>
+                    <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
+                    <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
+                    <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
+                </ul>
+            </div>
+        </div>
+        <button class="but1">Get my resume</button>
```

Comment:
- [ ] Kindly update the styles on this button to make it match the one on Figma. this includes the background color and the interactions with this button and all the other buttons (hover, pressed...)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192100642-c529075c-be01-4cda-9182-1b1832d9f0d7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192100653-72e2a652-ae3f-4a5e-8787-b4ef874f8f25.png)
![image](https://user-images.githubusercontent.com/50721479/192100657-6b884233-e5bc-4845-9678-65a64ee05312.png)